server/registry/actions_status_test.go

```diff
@@ -38,7 +38,13 @@ func TestGetStatus(t *testing.T) {
 		t.Fatalf("GetStatus(%+v) returned error: %s", req, err)
 	}
 
-	if !cmp.Equal(want, got, protocmp.Transform()) {
+	opts := cmp.Options{
+		protocmp.Transform(),
+		// Ignore fields that are build-dependent or excluded from Go 	< 1.18.
+		protocmp.IgnoreFields(&rpc.Status{}, "build"),
+	}
+
+	if !cmp.Equal(want, got, opts) {
 		t.Errorf("GetStatus(%+v) returned unexpected diff (-want +got):\n%s", req, cmp.Diff(want, got, protocmp.Transform()))
```

Comment:
Did you intentionally use different options for the diff and equal? IMO they should use the same options.

Reply:
No, thanks for catching that. Will fix.